### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by OmarMWarraich**
This is the first capstone project in the Microverse curriculum.
This website has been made by me in vanilla Javascript and Html.
I have created a fully responsive website.
Media Query has been applied at the break point of 768 pixels.
The video link showing the UI is as follows:-
https://www.loom.com/share/fe46095e60374fe39ae254665000e734?t=302

The updated video link is as follows:
https://www.loom.com/share/7d325f3abd3e455da78c4b8e17fb9c50

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @OmarMWarraich, Kindly fix Stylelint errors on your pr 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @OmarMWarraich, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used



## Required Changes ♻️
- [ ] Good presentation 👏🏿 Your presentation should also contain a highlight about a piece of the code you find  interesting or you are  proud of making based on the [project documentation](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#project-documentation) requirements, please record another video following all the requirements 🤗 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+        </div>
+        <div class="nav-body">
+            <div class="nav_logo">    
+                <a href="index.html" class="nav-logo">
+                    <img src="./assets/O-va_logo.png" alt="logo">
+                </a>
+            </div>
+            <div class="nav-menu">
+                <ul class="nav-list">
+                    <li class="nav-item1"><a href="#">About</a></li>
```
[the remark this answers is not in the document]
Comment (OmarMWarraich) [reply about.html]: image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+          <div class="main-content1">
+              <div class="main-content-title">
+                  <h1>"Hello! Techno world"</h1>
+              </div>
+              <div class="main-content-description">
+                  <h1>
+                      TECH WORLD <br/>
+                      GLOBAL SUMMIT <br/>
+                      2022
+                  </h1>
```
[the remark this answers is not in the document]
Comment (OmarMWarraich) [reply about.html]: image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+                <div class="main-content-title">
+                    <h1>"Hello! Techno world"</h1>
+                </div>
+
+                <div class="main-content-description">
+                    <h1>
+                        TECH WORLD <br/>
+                        GLOBAL SUMMIT <br/>
+                        2022
+                    </h1>
```
[the remark this answers is not in the document]
Comment (OmarMWarraich) [reply index.html]: image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+              <div class="toggle_icon1">
+                  <span></span>
+                  <span></span>
+                  <span></span>
+              </div>
+          </div>
+          <div>
+              <ul class="menu1">
+                  <li><a class="menu-items" href="./index.html">logo</a></li>
+                  <li><a class="menu-items" href="#">About</a></li>
```
[the remark this answers is not in the document]
Comment (OmarMWarraich) [reply about.html]: image

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @OmarMWarraich , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**